File: DJRSwiftResource/DJRSwiftResource/Classes/Common/UIFont+ex.m
//
//  UIFont+ex.m
//  LXToolKitObjC_Example
//
//  Created by lxthyme on 2023/4/19.
//

#import "UIFont+ex.h"

@implementation UIFont (ex)

+ (UIFont *)misansFontOfSize:(CGFloat)size {
    UIFont *font = [UIFont fontWithName:@"misans" size:size];
    if(!font) {
        font = [UIFont boldSystemFontOfSize:size];
    }
    return font;
}
@end

File: Example/Daily/2020/12-28/Extension/UIFont + ex.swift
//
//  UIFont + ex.swift
//  LXToolKit_Exam
//
//  Created by LXThyme Jason on 2021/1/9.
//  Copyright © 2021 CocoaPods. All rights reserved.
//

import Foundation
import UIKit

enum LXFontType: String {
    case light = "Light"
    case medium = "Medium"
    case regular = "Regular"
    case bold = "Bold"
    case semibold = "Semibold"
}

// MARK: - 👀PingFangSC Font
extension UIFont {
    convenience init?(PingFang size: CGFloat, type: LXFontType = .regular) {
        self.init(name: "PingFangSC-\(type.rawValue)", size: size)
    }
    convenience init?(CorsivaHebrew size: CGFloat, type: LXFontType) {
        self.init(name: "CorsivaHebrew-\(type.rawValue)", size: size)
    }
}
